benchmark/common.js

```diff
@@ -128,6 +128,11 @@ Benchmark.prototype.http = function(options, cb) {
 
 Benchmark.prototype._run = function() {
   const self = this;
+  sendResult({
```

Comment:
You should check that this works when running benchmarks individually (for example `node benchmark/url/url-format.js`), I don't think it will.